packages/react-components/src/templates/NewsDetailsPage.tsx

```diff
@@ -21,18 +21,20 @@ const richTextContainer = css({
   marginBottom: `${24 / perRem}em`,
 });
 
-type NewsDetailsPageProps = Pick<
-  NewsResponse,
-  'text' | 'title' | 'type' | 'created' | 'link' | 'linkText'
->;
+type NewsDetailsPageProps =
+  | Pick<
+      NewsResponse,
+      'text' | 'title' | 'type' | 'created' | 'link' | 'linkText'
+    >
+  | Pick<TutorialsResponse, 'text' | 'title' | 'created' | 'link' | 'linkText'>;
 
 const NewsDetailsPage: React.FC<NewsDetailsPageProps> = ({
   text = '',
   created,
   title,
-  type,
   link,
   linkText,
+  ...rest
```

Comment:
Another way around this could be 
```suggestion
  type = 'Tutorial'
```

Reply:
TS complains if I try to pass this type in the middle of the other props, this `rest` is not that good, but it was the only way I could make TS stop complaining

<img width="1001" alt="Screen Shot 2022-10-07 at 13 04 01" src="https://user-images.githubusercontent.com/16595804/194598152-c6022c55-1acf-4171-a8e2-8c21b97a5f6d.png">